client/src/components/PublicPages/AuditPlanner/ElectionResultsCard.tsx

```diff
@@ -382,11 +383,12 @@ const ElectionResultsCard: React.FC<IProps> = ({
             <tr>
               <td>
                 <Button
+                  aria-label={isMobileWidth ? 'Add Candidate' : undefined}
                   disabled={!editable}
                   icon="plus"
                   onClick={() => addCandidate(constructNewCandidate())}
                 >
-                  Add Candidate
+                  {(isTabletWidth || isDesktopWidth) && 'Add Candidate'}
```

Comment:
As noted in the PR description, there's still room for improvement, e.g. preventing form elements from hopping around when error strings exceed two lines, labels wrapping on really small screens, etc. But this PR covers the basics